.github/workflows/agnosticv-operator-publish.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: agnosticv-operator-publish
+on:
+  push:
+    branches-ignore:
+    - '*'
+    tags:
+    - 'agnosticv-operator-v[0-9]*'
+jobs:
+  publish:
+    env:
+      IMAGE_NAME: babylon-agnosticv-operator
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v2
+
+    - name: Get image tags
+      id: image_tags
+      run: |
+        # Version is a semantic version tag or semantic version with release number
+        # GITHUB_REF will be of the form "refs/tags/admin-v0.1.2" or "refs/tags/admin-v0.1.2-1"
+        # To determine RELEASE, strip off the leading "refs/tags/"
+        RELEASE=${GITHUB_REF#refs/tags/agnosticv-operator-}
+        # To determine VERSION, strip off any release number suffix
+        VERSION=${RELEASE/-*/}
+        echo "::set-output name=RELEASE::${RELEASE}"
```

Comment:
@jkupferer we have to update the output, it will be deprecated soon.
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

```
        echo "RELEASE=${RELEASE}" >> $GITHUB_OUTPUT
        echo "VERSION=${VERSION}" >> $GITHUB_OUTPUT
```

Reply:
Thanks @marcosmamorim , good catch. We'll need to update this in a lot of places!